ext/native-decls/GetGamePool.md

```diff
@@ -21,6 +21,16 @@ follows:
 * `CVehicle`: Vehicles.
 * `CPickup`: Pickups.
 
+## Examples
+```lua
+local vehiclePool = GetGamePool('CVehicle') -- Get the list of vehicles (entities) from the pool
+for _, veh in ipairs(vehiclePool) do -- loop through each vehicle (entity)
```

Comment:
```suggestion
for i = 1, #vehiclePool do -- loop for how many vehicles are found in vehiclePool
```
Isn't this more performant?

Reply:
performance should always be measured :)

Reply:
You can see a bunch of performance tests done [here](https://springrts.com/wiki/Lua_Performance#TEST_9:_for-loops) from forever ago

Reply:
I overlooked this, thanks for noticing. According to the following sources doing this without `pairs` or `ipairs` is faster yes.

https://medium.com/@sleitnick/battle-of-the-loops-c001bcb4961c
https://stackoverflow.com/a/8955122 (this answer sources information from @AvarianKnight's link it seems)

I will be correcting this PR in a bit.

Reply:
> Isn't this more performant?

Updated the example code to include your suggestion and squashed my commits, thanks :-)